source/strings.tex

```diff
@@ -4661,8 +4661,8 @@
 
 \pnum
 \begin{note}
-The library provides implicit conversions from \tcode{const charT*} and \tcode{std::basic_string<charT, ...>} to \tcode{std::basic_string_view<charT, ...>} so that user code can accept just \tcode{std::basic_string_view<charT>} as a non-templated parameter wherever a sequence of characters is expected.
-User-defined types should define their own implicit conversions to \tcode{std::basic_string_view} in order to interoperate with these functions.
+The library provides implicit conversions from \tcode{const charT*} and \tcode{basic_string<charT, ...>} to \tcode{basic_string_view<charT, ...>} so that user code can accept just \tcode{basic_string_view<charT>} as a non-templated parameter wherever a sequence of characters is expected.
+User-defined types should define their own implicit conversions to \tcode{basic_string_view} in order to interoperate with these functions.
 \end{note}
```

Comment:
Hm... This is a note explaining how to interoperate with string views in user code. Assuming no "using namespace std" is present, user code would need the `std::` prefix when mentioning `basic_string_view`.

On the other hand, talking about `basic_string_view<charT>` as a "non-templated parameter" certainly means this is not verbatim user code (because `charT` is used as a template parameter in this section).

Net result: I'm fine with the proposed change.